css/main.scss

```diff
@@ -180,7 +180,30 @@ footer .footer-block {
 }
 
 
-
-
-
-
+.answers .answer {
+  display: none;
+  &.default-open {
+    display: block;
+  }
+  &:target {
+    display: block;
+    & ~ .answer {
+      display: none !important;
+    }
+  }
+}
+.answer {
```

Comment:
This makes it so that when you navigate to the anchor, the top of the element isn't hidden underneath the fixed navbar